scipy/optimize/_lbfgsb_py.py

```diff
@@ -279,24 +279,25 @@ def _minimize_lbfgsb(fun, x0, args=(), jac=None, bounds=None,
     x0 = asarray(x0).ravel()
     n, = x0.shape
 
+    # historically old-style bounds were/are expected by lbfgsb.
+    # That's still the case but we'll deal with new-style from here on,
+    # it's easier
```

Comment:
Since a lot of things age, could you explicitly define what is "old-style" and "new-style" here? Thank you. :pray:

Reply:
Its known within scipy what old and new style are.

Reply:
It's also outlined in the docstring of `old_bound_to_new`